ui/admin/app/components/form/field/list-wrapper/select/index.hbs

```diff
@@ -0,0 +1,79 @@
+<Hds::Table class='list-wrapper-field' name={{@name}}>
+
+  <:body as |B|>
+
+    {{#each this.options as |select index|}}
+      <B.Tr>
+        <B.Td>
+          <Hds::Form::Select::Field
```

Comment:
Should we be using `<Hds::Form::Select::Base` since we're not using any of the labels or helper text?

Reply:
@ZedLi yes good catch!

Reply:
@ZedLi added in latest commit